pack/parallel/rod_encounter.json

```diff
@@ -0,0 +1,77 @@
+[
+    {
+        "back_text": "<b>Hard / Expert</b>\n[skull]: -X. X is 1 more than the number of [[Tome]] assets Daisy Walker controls.\n[cultist]: Reveal another token. If you fail, discard the top 3 cards of your deck.\n[tablet]: -3. If this is an attack or evasion attempt against Namer of the Dead and you do not succeed by at least 3, it attacks you.\n[elder_thing]: -5. If you fail, Daisy Walker takes 1 horror.",
```

Comment:
No need for bold tags around 'hard / expert' (we changed style halfway through Dunwich - I need to remove the bold tags from the first few scenario cards)